modules/tensor_mechanics/doc/content/documentation/systems/Materials/ComputeMultipleInelasticStress.md

```diff
@@ -1,14 +1,242 @@
+$i^{th}$ inelastic material model, represented by the blue element [fig:multiple_materials].
+The inelastic model calculates the inelastic strain increment necessary to produce
+an admissible stress, as determined by the model's yield function. The specifics
+of the individual inelastic model admissible stress criteria are given in the
+documentation for the inelastic model. The elastic and inelastic strain increments,
+stress, and, optionally, the consistent tangentoperator are returned to
+`ComputeMultipleInelasticStress` from the $i^{th}$ inelastic material model.
+
+### Outer Iteration over Stress Difference
+
+After the inner iteration over all the inelastic material models is complete,
+the maximum and minimun stress values ($\sigma_{max}$ and $\sigma_{min}$) are
+identified from the set of stresses calculated by the individual inelastic models.
+An $L^2$ norm calculation is performed on the stress difference
+\begin{equation}
+  \label{eqn:l2_norm_stress}
+  L^2 \left(\Delta \sigma \right) = \sqrt{\Delta \sigma_{ij} \Delta \sigma_{ij}}
+    \text{, where } \Delta \sigma = \sigma_{max} - \sigma_{min}
+\end{equation}
+The $L^2$ norm of the stress difference is compared to the absolute and relative
+tolerances to determine if the solution from the combined inelastic material
+models is converged
+\begin{equation}
+  \label{eqn:l2_norm_convergence}
+  L^2 \left(\Delta \sigma \right) < \text{absolute tolerance, or }
+  \frac{L^2 \left(\Delta \sigma \right)}{L^2 \left(\Delta \sigma_o \right)} < \text{relative tolerance}
+\end{equation}
+where $L^2 \left(\Delta \sigma_o \right)$ is the $L^2$ norm from the very first
+outer iteration over all of the inelastic material models.
+The solution will not converge if the outer iteration loop, shown in the top half
+of [fig:multiple_materials], exceeds the maximum number of iterations set by the
+user.
+
+### Finalize Strains and the Jacobian Multiplier
+
+Once convergence on the stress is obtained, the method finalizes the calculation
+of the inelastic strains by applying a weighting factor, as shown in [fig:multiple_materials].
+This weighting factor has a default value of unity.
+
+The Jacobian multiplier, which is used in the [StressDivergenceTensors](Kernels/StressDivergenceTensors.md)
+kernel to condition the Jacobian calculation, must be calculated from the combination
+of all the different inelastic material models. There are three options used to
+calculate the combined Jacobian multiplier: Elastic, Partial, and Nonlinear.
+\begin{equation}
+  \label{eqn:combined_jacobian_mult}
+  \mathbf{J}_m = \begin{cases}
+                  \mathbf{C} & \text{Elastic option} \\
+                  \mathbf{A}^{-1} \cdot \mathbf{C} \text{, where }
+                      \mathbf{A} = \mathbf{I} + \sum_i \mathbf{C}^{to} & \text{Partial option} \\
+                  \prod_i \mathbf{C}^{-1} \cdot \mathbf{C}^{to} & \text{Nonlinear option}
```

Comment:
The math here goes:
if one model: C^to
if multiple models: C^to_3 * C^-1 * C^to_2 * C^-1 * C_to_1.

Reply:
Added a new equation to the single model section for this math and added the subscript `i` to this equation to clarify the product sum